src/main/java/com/philips/research/bombase/core/meta/MetaInteractor.java

```diff
@@ -82,7 +83,7 @@ public void createPackage(PackageURL purl) {
     public Map<String, AttributeDto> setAttributes(PackageURL purl, Map<String, @NullOr Object> values) {
         registry.edit(purl, pkg -> values.forEach((key, value) -> {
             final var field = Field.valueOf(Field.class, key.toUpperCase());
-            pkg.update(field, 100, value);
+            pkg.update(field, Trust.TRUTH, value);
```

Comment:
Is this the default?  `Trust.TRUTH`? Shouldn't this be MAYBE?

Reply:
The Interactor is where the actions of a _human_ user come in. My expectation is that humans want to set the "truth" after careful investigation (or checking alternatives from automated processes), and would not accept automated overrides by automated harvesters.